### PR TITLE
Mark invalid manifests in Secscan V4

### PR DESCRIPTION
**Pull request by josephschorr**
If we get an invalid manifest when indexing, we now mark the manifest
as invalid for security indexing

Fixes https://issues.redhat.com/browse/PROJQUAY-949

### CODE REVIEW

File: data/secscan_model/secscan_v4_model.py [hunk 1]
```diff
@@ -214,6 +214,28 @@ def needs_reindexing_query(indexer_hash):
         for candidate, abt, num_remaining in iterator:
             manifest = ManifestDataType.for_manifest(candidate, None)
             layers = registry_model.list_manifest_layers(manifest, self.storage, True)
+            if layers is None:
```
Comment (alecmerdler) [inline data/secscan_model/secscan_v4_model.py:217]: Should we add a test for this?

Comment (josephschorr) [reply data/secscan_model/secscan_v4_model.py:217]: Yes, we should, but I wanted to get this fixed for QE. Should I do so as a followup or would you like to do so?

Comment (alecmerdler) [reply data/secscan_model/secscan_v4_model.py:217]: We can do it as a followup.

Comment (josephschorr) [reply data/secscan_model/secscan_v4_model.py:217]: Added it to this PR

**Pull request merged**